internal/batches/service/service.go

```diff
@@ -43,10 +43,12 @@ func New(opts *Opts) *Service {
 // The reason we ask for batchChanges here is to surface errors about trying to use batch
 // changes in an unsupported environment sooner, since the version check is typically the
 // first thing we do.
-const sourcegraphVersionQuery = `query SourcegraphVersion {
+
```

Comment:
```suggestion

```

I promise this is actually relevant. 😜 The extra line between the docstring and the variable definition breaks inline documentation on hover for this variable. 😱

Without the extra new line:
<img width="558" alt="image" src="https://user-images.githubusercontent.com/8942601/215256589-0a0c485c-0b3d-451f-8e12-033401f5b04e.png">

With the extra new line:
<img width="555" alt="image" src="https://user-images.githubusercontent.com/8942601/215256601-7ae1859d-58fb-4143-b016-f366153f83f1.png">

Reply:
Ugh, okay, the diff view really makes it difficult to understand what my suggestion is, lol.

Currently you have this:
```go
// The reason we ask for batchChanges here is to surface errors about trying to use batch
// changes in an unsupported environment sooner, since the version check is typically the
// first thing we do.

const sourcegraphVersionQuery = `query SourcegraphVersion {
...
```

My suggestion here and in the following two cases is to remove the extra line between the comment and the variable, which preserve the inline documentation on hover:

```go
// The reason we ask for batchChanges here is to surface errors about trying to use batch
// changes in an unsupported environment sooner, since the version check is typically the
// first thing we do.
const sourcegraphVersionQuery = `query SourcegraphVersion {
...
```